spec/unit/enforcement_spec.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
+class MyController # < ApplicationController
+  include Blacklight::AccessControls::Enforcement
+end
+
 describe Blacklight::AccessControls::Enforcement do
-  let(:controller) { CatalogController.new }
-  let(:search_builder) { SearchBuilder.new(method_chain, controller) }
-  let(:method_chain) { SearchBuilder.default_processor_chain }
+  let(:controller) do
+    c = MyController.new
+    allow(c).to receive(:current_ability).and_return(ability)
```

Comment:
That doesn't do what you think it does.  The class is still visible everywhere after the parser gets to it.  It is a false sense of scoping.